scripts/src/admin/services.ts

```diff
@@ -562,6 +563,14 @@ export const updateAppConfig = async function(req: UpdateAppConfigRequest, res:
 	}
 
 	const app = (await Application.findOne({ id: req.params.application_id }))!;
+
+	// if the change is migrating from v2 to v3, enforce gradual migration
+	if (app.config.blockchain_version === "2" && config.blockchain_version === "3"){
+		config.gradual_migration_date = moment().toISOString();
+		config.limits.hourly_migration = 1;
```

Comment:
You don't really want to add these values to production - we should get it from the config

Reply:
done